library/src/main/java/com/google/android/exoplayer/text/eia608/ClosedCaptionCtrl.java

```diff
@@ -69,6 +71,45 @@
   public static final byte TAB_OFFSET_CHAN_1 = 0x17;
   public static final byte TAB_OFFSET_CHAN_2 = 0x1F;
 
+  private static final String TAG = "Eia608CaptionControl";
+
+  private static final int[] COLOR_VALUES = {
+          Color.WHITE,
+          Color.GREEN,
+          Color.BLUE,
+          Color.CYAN,
+          Color.RED,
+          Color.YELLOW,
+          Color.MAGENTA,
+          Color.TRANSPARENT // Last color setting should be kept
+  };
+
+  // for debug purposes
```

Comment:
I added getIndentation() while I was trying to figure out the meaning of the bits of the stream. Although it is not useful without full positioning, it might help anyone how is attempting to update the code. 
Now, when you stop at a break point, Android Studio is able to write out every piece of information about the incoming Commands instead of a single HEX value. Shall I remove it? (Either ways I will add most of it back with the next change set implementing correct positioning).